woocommerce/woocommerce
# Add image gallery component

### All Submissions:






### Changes proposed in this Pull Request:

This PR adds the image gallery component.  In order to do this it also:
* Refactors the `Sortable` component to allow sorting vertically
* Extracts the `ListItem` component to reduce opinionated styling of sortable items

Note that the toolbar for the Image Gallery will be a follow-up in another PR.  This one focuses on base gallery styling and sortability.


<img width="375" alt="Screen Shot 2022-08-23 at 12 46 37 PM" src="https://user-images.githubusercontent.com/10561050/186252352-a2e6ca23-3286-486a-a138-6509e6874eca.png">
<img width="340" alt="Screen Shot 2022-08-23 at 12 34 27 PM" src="https://user-images.githubusercontent.com/10561050/186252355-af7f72d6-3e7e-4d52-aaf6-bf4fdf2f7bc3.png">
<img width="673" alt="Screen Shot 2022-08-23 at 12 34 21 PM" src="https://user-images.githubusercontent.com/10561050/186252359-22e20aaa-9886-40bd-a4f4-8555556bfdaa.png">

### How to test the changes in this Pull Request:

1. `npm run storybook`
2. View the `Sortable`, `ImageGallery`, and `ListItem` stories
3. Check that things work as expected

### Other information:




### FOR PR REVIEWER ONLY:
